deployment/sia-dev-gw-in-ppe-cluster.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: sia-dev-gw-svc
```

Comment:
Is it possible to rename to `sia-dev-gateway-service`? (I prefer longer, more descriptive names and variables.)